media/css/mozorg/contribute.scss

```diff
@@ -110,10 +158,26 @@ $image-path: '/media/protocol/img';
         .contribute-card-wrapper {
             display: flex;
             flex-direction: column;
+            max-width: 100%;
         }
 
         .contribute-card-content {
             flex: 1;
+
+            // fallback when hyphens aren't supported
+            overflow-wrap: break-word;
+            word-wrap: break-word; // IE specific overflow-wrap fix
+        }
+
+        @supports (hyphens: auto) {
+            .contribute-card-content {
+                overflow-wrap: unset;
+                word-wrap: unset;
+            }
+
+            .mzp-c-card-title {
```

Comment:
<img width="1252" alt="headline-hyphens" src="https://user-images.githubusercontent.com/19650432/200805150-5f358e73-a051-4f58-aba4-bbc430b64e4f.png">

note: this will still use the break-word behaviour on browsers where hyphens are not supported.

Other option I can think of is allowing the content to wrap to two rows when the flex-basis space is smaller than min-content of card.